integration/kubernetes/confidential/sev.bats

```diff
@@ -0,0 +1,323 @@
+  echo "-------------------------------------------------------------------------------"
+  esudo kubectl get services -o wide
+  echo "-------------------------------------------------------------------------------"
+  esudo kubectl get deployments -o wide
+  echo "-------------------------------------------------------------------------------"
+  esudo kubectl get pods -o wide
+  echo "-------------------------------------------------------------------------------"
+  pod_name=$(esudo kubectl get pod -o wide | grep encrypted-image-tests | awk '{print $1;}')
+  esudo kubectl describe pod ${pod_name}
+  echo "-------------------------------------------------------------------------------"
+}
+
+setup_file() {
+  echo "###############################################################################"
+  echo -e "SETUP FILE - STARTED\n"
+
+  start_date=$(date +"%Y-%m-%d %H:%M:%S")
+
+  TEST_DIR="$(mktemp -d /tmp/test.XXXXXXXX)"
+
+  # Install package dependencies
+  echo "Installing required packages..."
+  esudo apt install -y docker-compose jq
+  pip install sev-snp-measure
+
+  # KBS setup and run
+  echo "Setting up simple-kbs..."
+  run_kbs
+
+  # Pull image and retrieve encryption and ssh keys
+  echo "Pulling encrypted image and setting keys..."
+  pull_encrypted_image_and_set_keys
+
+  echo "SETUP FILE - COMPLETE"
+  echo "###############################################################################"
+}
+
+setup() {
+  # Remove the service/deployment/pod if it exists
+  echo "Deleting previous test pods..."
+  esudo kubectl delete -f \
+    "${TESTS_REPO_DIR}/integration/kubernetes/confidential/fixtures/unencrypted-image-tests.yaml" 2>/dev/null || true
+  esudo kubectl delete -f \
+    "${TESTS_REPO_DIR}/integration/kubernetes/confidential/fixtures/encrypted-image-tests.yaml" 2>/dev/null || true
+
+  # Delete any previous data in the DB
+  mysql -u${KBS_DB_USER} -p${KBS_DB_PW} -h ${KBS_DB_HOST} -D ${KBS_DB} <<EOF
+    DELETE FROM secrets WHERE id = 10;
+    DELETE FROM keysets WHERE id = 10;
+    DELETE FROM policy WHERE id = 10;
+EOF
+}
+
+@test "$test_tag Test SEV unencrypted container launch success" {
+  # Add key to KBS with policy measurement
+  add_key_to_kbs_db
+  
+  # Start the pod and retrieve the name
+  esudo kubectl apply -f \
+    "${TESTS_REPO_DIR}/integration/kubernetes/confidential/fixtures/unencrypted-image-tests.yaml"
+  sleep 20
+
+  print_service_info
+  pod_ip=$(esudo kubectl get pod -o wide | grep encrypted-image-tests | awk '{print $6;}')
+
+  # Look for SEV enabled in container dmesg output
+  sev_enabled=$(ssh -i ${TEST_DIR}/encrypted-image-tests \
+    -o "StrictHostKeyChecking no" \
+    -o "PasswordAuthentication=no" \
+    -t root@${pod_ip} \
+    'dmesg | grep SEV' || true)
+
+  if [ -z "$sev_enabled" ]; then
+    >&2 echo -e "${RED}KATA CC TEST - FAIL: SEV is NOT Enabled${NC}"
+    return 1
+  else
+    echo "DMESG REPORT: $sev_enabled"
+    echo -e "${GREEN}KATA CC TEST - PASS: SEV is Enabled${NC}"
```

Comment:
Kata has a test for TDX which checks that once confidential guest is enabled then the guest memory is encrypted. So could we add here a similar check? Don´t  need to be on this PR...

The check implementation for TDX: https://github.com/kata-containers/tests/blob/CCv0/functional/tdx/run.sh#L50 . It would be made into a shared function.